### PR TITLE
Revamp organization dropdown in the Console header

### PR DESCRIPTION
**Pull request by NipuniBhagya**
### Related Issues
- Issue `#1` or (None)

### Related PRs
- Related PR `#1` or (None)

### Checklist
- [ ] e2e cypress tests locally verified.
- [ ] Manual test round performed and verified.
- [ ] UX/UI review done on the final implementation.
- [ ] Documentation provided. (Add links if there are any)
- [ ] Unit tests provided. (Add links if there are any)
- [ ] Integration tests provided. (Add links if there are any)

### Security checks
- [ ] Followed secure coding standards in http://wso2.com/technical-reports/wso2-secure-engineering-guidelines?
- [ ] Ran FindSecurityBugs plugin and verified report?
- [ ] Confirmed that this PR doesn't commit any keys, passwords, tokens, usernames, or other secrets?

### CODE REVIEW

File: apps/console/src/features/organizations/components/organization-switch/organization-switch-breadcrumb.tsx [hunk 1]
```diff
@@ -342,70 +320,45 @@ export const OrganizationSwitchBreadcrumb: FunctionComponent<OrganizationSwitchD
...
+            <>
+                {
+                    !isLoading && (
+                        <div className="organization-breadcrumb-wrapper">
+                            <div 
+                                className="organization-breadcrumb" 
+                                onClick={ () => setIsDropDownOpen(!isDropDownOpen) }
+                            >
+                                <p className="organization-breadcrumb-label">
+                                    Organization
```
Comment (thivi) [inline apps/console/src/features/organizations/components/organization-switch/organization-switch-breadcrumb.tsx:334]: Shall we localize this?

File: apps/console/src/features/organizations/components/organization-switch/organization-switch-breadcrumb.tsx [hunk 2]
```diff
@@ -151,137 +148,129 @@ export const OrganizationSwitchBreadcrumb: FunctionComponent<OrganizationSwitchD
...
-                                            <Icon
-                                                name="angle right"
-                                                className="separator-icon"
-                                            />
-                                        ) }
-                                    </Menu.Item>
+                                        </Breadcrumb.Section>
+                                        <Breadcrumb.Divider 
+                                            style={ { 
+                                                "display": index === breadcrumbList.length - 1 ? "none" : "inline-block"
```
Comment (thivi) [inline apps/console/src/features/organizations/components/organization-switch/organization-switch-breadcrumb.tsx:208]: Shall we use classes for this?

File: apps/console/src/features/organizations/components/organization-switch/organization-switch-breadcrumb.tsx [hunk 3]
```diff
@@ -151,137 +148,129 @@ export const OrganizationSwitchBreadcrumb: FunctionComponent<OrganizationSwitchD
...
-                                                    : "ellipsis"
+                                                    ? "organization-breadcrumb-item un-clickable ellipsis"
+                                                    : "organization-breadcrumb-item ellipsis"
                                             }
-                                            data-componentid={ `${ componentId }-breadcrumb-item-${ breadcrumb.name }` }
+                                            data-componentid={ 
+                                                `${ componentId }-breadcrumb-item-${ breadcrumb.name }` 
+                                            }
+                                            style={ {
+                                                "pointer-events": index === breadcrumbList.length - 1 ? "none" : "auto"
```
Comment (thivi) [inline apps/console/src/features/organizations/components/organization-switch/organization-switch-breadcrumb.tsx:200]: Shall we use classes for this?

File: apps/console/src/features/organizations/components/organization-switch/organization-switch-breadcrumb.tsx [hunk 4]
```diff
@@ -132,13 +127,15 @@ export const OrganizationSwitchBreadcrumb: FunctionComponent<OrganizationSwitchD
...
-        item: BreadcrumbItem
+        item?: BreadcrumbItem
     ): ReactElement => {
         return OrganizationUtils.isRootOrganization(item) ? (
-            organizationConfigs.superOrganizationBreadcrumb(
-                item,
-                handleOrganizationSwitch
-            )
+            <Breadcrumb.Section 
+                style={ { "color": "#000000" } }
```
Comment (thivi) [inline apps/console/src/features/organizations/components/organization-switch/organization-switch-breadcrumb.tsx:134]: Shall we use a class for this?

**Review by savindi7 — DISMISSED** [review]

File: apps/console/src/features/organizations/components/organization-switch/organization-switch-breadcrumb.tsx [hunk 5]
```diff
@@ -132,21 +123,80 @@ export const OrganizationSwitchBreadcrumb: FunctionComponent<OrganizationSwitchD
...
+                                    breadcrumbList.length <= 2 && (
+                                        <Icon
+                                            key={ index }
+                                            claslName="organization-breadcrumb-icon"
+                                            name={ isDropDownOpen ? "angle up" : "angle down" }
+                                            className="separator-icon"
+                                        /> 
+                                    )
+                                }
+                            </>
```
Comment (thivi) [inline apps/console/src/features/organizations/components/organization-switch/organization-switch-breadcrumb.tsx:196]: Do we need the fragments here?

**Review by savindi7 — APPROVED** [review]

**Review by thivi — APPROVED** [review]

**Pull request merged**